README.md

```diff
@@ -1,36 +1,265 @@
+
+Insert into the top of the PCB and solder. [Just like step 7 of the MurphPad build guide](https://mechwild.com/guides/build-guides/murphpad-build-guide/#step7). You may need to bend some pins back if they were bent out of place during shipment.
+
+### Step 7: Plug In and Test
+
+If you didn't already attach the Pro Micro to the PCB in step 4, do it now.
+
+1. Plug the board in. If the OLED (and optionally RGB strip) light up, congratulations! :tada:
+2. Use metal tweezers or a bent paper clip to check each switch position.
+   - If you used an encoder, twist *and* press down on it.
+
+#### Testing in Vial and VIA
+
+[Vial](https://get.vial.today) and [VIA](https://usevia.app/) have dedicated matrix testing modes that can be enabled by [sideloading the BDE Rev2 layout definition](https://docs.cannonkeys.com/via-json-loading/). This mode takes a lot of guesswork out of the testing process.
+
+| Program | Screenshot |
+| ---- | ---- |
+| Vial | ![Test Matrix mode in Vial](https://cdn.discordapp.com/attachments/837441710698004531/886197994100240394/unknown.png) |
+| VIA  | ![Test Matrix mode in VIA](https://cdn.discordapp.com/attachments/837441710698004531/885969317743718400/unknown.png) |
+
+The encoder turn won't show on the matrix, but the press should.
+
+#### Testing in QMK Configurator
+
+Load [QMK Configurator with the expected keymap](https://config.qmk.fm/) in one tab and [QMK Configurator in test mode](https://config.qmk.fm/#/test) in another, and make sure all the keys do what the keymap expects. Keep an eye out for layer keys.
+
+### Step 8X: Install 0305/7305 Mill-Max Sockets
+
+If you did not purchase these sockets, [skip to step 9](#step-9-install-switches).
+
+#### ...by taping them to the PCB
+
+1. Insert one Mill-Max socket into each switch pin hole. The lip of the socket sits on the top of the PCB, while the rest falls through the hole.
+2. Place painter's tape, masking tape or polyimide ("Kapton") tape on top of the PCB, to hold the sockets in place while the PCB is upside down.
+   - Electrical tape will also work, but may leave an unpleasant residue.
+   - Scotch tape is not recommended.
+3. Flip the PCB, and solder each socket into place.
+
+#### ...by inserting switches into the PCB
+
+[Follow these instructions](https://docs.splitkb.com/hc/en-us/articles/360012930919-How-do-I-install-Mill-Max-Hot-Swap-Sockets-).
+
+### Step 9: Install Switches
+
+1. Insert four switches into the top (switch) plate, one in or near each corner. These should click into place.
+2. Place this plate + switch assembly onto the PCB. Verify all eight switch pins are sticking through the holes in the PCB.
+3. Continue inserting switches into the plate, making sure both pins on each switch poke through the holes in the PCB. It may be necessary to hold the PCB in place to prevent it from being pushed off, especially in the beginning.
+
+If you did not install Mill-Max sockets earlier, you will need to solder the switches after putting them in place.
```

Comment:
If you did install mill-max sockets, be careful when inserting to make sure that both pins are aligned with the sockets before pushing the socket into place. On switches with thinner pins, you may not be able to feel the difference between a switch that has correctly entered the mill-max sockets and a switch where one or both pins have bent upon being forced downwards.

It can be helpful to balance the PCB and plate perpendicular to your work surface, then work from the bottom rows up and looking between the PCB and plate to verify pin alignment.

Reply:
There's way too much sense in this advice to be useful for me 😆